### PR TITLE
DMLC link removal

### PR DESCRIPTION
**Pull request by IvyBazan**
Replaced v1.5.x Julia page with the content on master. Also, replaced dmlc.ml links with available content.

### CODE REVIEW

**Comment by piyushghai** [comment]
@mxnet-label-bot Add [Doc]

**Comment by IvyBazan** [comment]
Preview: http://100.24.115.165/api/julia/index.html

**Comment by IvyBazan** [comment]
@aaronmarkham

File: tools/coreml/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<--- Licensed to the Apache Software Foundation (ASF) under one -->
```
Comment (aaronmarkham) [inline tools/coreml/README.md]: Accidental edit?

File: example/neural-style/end_to_end/README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 # End to End Neural Art
 
-Please refer to this [blog](http://dmlc.ml/mxnet/2016/06/20/end-to-end-neural-style.html) for details of how it is implemented.
+Please refer to this [blog](#) for details of how it is implemented.
```
Comment (aaronmarkham) [inline example/neural-style/end_to_end/README.md]: Maybe use this one? https://thomasdelteil.github.io/NeuralStyleTransfer_MXNet/

File: docs/faq/bucketing.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ This approach works with variable length sequences. For more complicated models
 
 ## Variable-length Sequence Training for Sherlock Holmes
 
-We use the [Sherlock Holmes language model example](https://github.com/dmlc/mxnet/tree/master/example/rnn) for this example. If you are not familiar with this example, see [this tutorial (in Julia)](http://dmlc.ml/mxnet/2015/11/15/char-lstm-in-julia.html) first.
+We use the [Sherlock Holmes language model example](https://github.com/dmlc/mxnet/tree/master/example/rnn) for this example. If you are not familiar with this example, see [this tutorial (in Julia)](https://mxnet.incubator.apache.org/versions/master/api/julia/site/tutorial/char-lstm/) first.
```
Comment (marcoabreu) [inline docs/faq/bucketing.md]: please only use relative links and not versioned ones

File: tools/coreml/test/test_mxnet_models.py [hunk 4]
```diff
@@ -151,8 +151,8 @@ def test_pred_vgg16(self):
 
     def test_pred_nin(self):
         self._test_model(model_name='nin', epoch_num=0,
-                         files=["http://data.dmlc.ml/models/imagenet/nin/nin-symbol.json",
-                                "http://data.dmlc.ml/models/imagenet/nin/nin-0000.params"])
+                         files=["",
```
Comment (marcoabreu) [inline tools/coreml/test/test_mxnet_models.py]: ???

File: tools/coreml/README.md [hunk 5]
```diff
@@ -100,7 +100,7 @@ Any MXNet model that uses the above operators can be converted easily. For insta
 mxnet_coreml_converter.py --model-prefix='Inception-BN' --epoch=126 --input-shape='{"data":"3,224,224"}' --mode=classifier --pre-processing-arguments='{"image_input_names":"data"}' --class-labels synset.txt --output-file="InceptionBN.mlmodel"
 ```
 
-2. [NiN](http://data.dmlc.ml/models/imagenet/nin/)
+2. [NiN](#)
```
Comment (marcoabreu) [inline tools/coreml/README.md]: ?

File: docs/tutorials/tensorrt/inference_with_trt.md [hunk 6]
```diff
@@ -118,7 +118,7 @@ for i in range(0, 10000):
 end = time.time()
 print(time.process_time() - start)
 ```
-We run timing with a warmup once more, and on the same machine, run in **18.99s**. A 1.8x speed improvement!  Speed improvements when using libraries like TensorRT can come from a variety of optimizations, but in this case our speedups are coming from a technique known as [operator fusion](http://dmlc.ml/2016/11/21/fusion-and-runtime-compilation-for-nnvm-and-tinyflow.html).
+We run timing with a warmup once more, and on the same machine, run in **18.99s**. A 1.8x speed improvement!  Speed improvements when using libraries like TensorRT can come from a variety of optimizations, but in this case our speedups are coming from a technique known as [operator fusion](#).
```
Comment (marcoabreu) [inline docs/tutorials/tensorrt/inference_with_trt.md]: ?

**Comment by aaronmarkham** [comment]
@marcoabreu Sorry about rushing that through without addressing your questions. We needed to be rid of the malware links ASAP. 
@IvyBazan Can you answer? We spoke about this offline and you had some reasoning why it might be ok to just kill the links with a #.

**Comment by IvyBazan** [comment]
@marcoabreu I removed those links because I couldn't find any replacement for the content that was hosted on dmlc.ml. It seems better to temporarily have replaced the links with a #, while waiting for that content to be recovered and rehosted, rather than having malware links still active on the site. The current experience is that the links redirect to the current page.

**Comment by IvyBazan** [comment]
Issues related to missing content:
Missing nin content-https://github.com/apache/incubator-mxnet/issues/15729#issuecomment-518131446
Missing nnvm blog: https://github.com/apache/incubator-mxnet/issues/15730#issuecomment-518131577

**Pull request merged**